pyrobosim/pyrobosim/core/locations.py

```diff
@@ -17,22 +18,26 @@ class Location:
     """Representation of a location in the world."""
 
     # Default class attributes
-    metadata = EntityMetadata()
+    metadata = EntityMetadata(metadata_type="locations")
     """ Metadata for location categories. """
     height = 1.0
     """ Vertical height of location. """
     viz_color = (0, 0, 0)
     """ Visualization color (RGB tuple). """
 
     @classmethod
-    def set_metadata(cls, filename):
+    def add_metadata(cls, filename):
```

Comment:
Maybe instead of replacing `set_metadata()`, we can keep that around for compatibility, and simply add a new `add_metadata()`  method that is additive instead of replaces?

Reply:
I updated the logic.